src/services/zapper.ts

```diff
@@ -104,6 +105,29 @@ export class ZapperService extends Service {
     return balances;
   }
 
+  /**
+   * Fetches vault token market data for the Yearn application
+   * @returns list of vault token market data
+   */
+  async tokenMarketData(): Promise<VaultTokenMarketData[]> {
```

Comment:
Since for now we mostly need supported vaults addresses, wdyt if we rename to `supportedVaults`, and just return `Address[]` ?

Reply:
I think it's a good idea, makes it much simpler as well https://github.com/yearn/yearn-sdk/pull/276/commits/11fc7003631938dc127aa339b56620a92bc72d16